lib/membrane_h264_ffmpeg/encoder.ex

```diff
@@ -104,16 +106,16 @@ defmodule Membrane.H264.FFmpeg.Encoder do
   @impl true
   def handle_process(:input, buffer, _ctx, state) do
     %{encoder_ref: encoder_ref, use_shm?: use_shm?} = state
-    pts = buffer.pts || 0
+    pts = if is_nil(buffer.pts), do: @no_pts, else: Common.to_h264_time_base_truncated(buffer.pts)
```

Comment:
Such `if` repeats both here and in the encoder, you can handle that in `Common` as well. That would make my previous comment obsolete, as `@no_pts` would not be needed outside `Common` module.
```suggestion
    pts = Common.to_h264_time_base_truncated(buffer.pts)
```